pipeline/lib/pipeline.ml

```diff
@@ -130,7 +130,8 @@ let pipeline ~ocluster ~conninfo ~repository env =
         Logs.err (fun log -> log "Error in %s stage: %s\n\n" stage m);
         Storage.record_stage_failure ~stage ~serial_id ~reason:m ~conninfo
     | _ -> ()
-  in
+  and+ () = ocluster_worker
```

Comment:
This looks good to me! Just to confirm, we bind the cluster worker and output so that we only send the state to Github once we have the output right? :)

Reply:
Yes exactly!